concrete/routes/boards.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+defined('C5_EXECUTE') or die('Access Denied.');
+/**
+ * @var \Concrete\Core\Routing\Router
+ */
+$router->all('/ccm/system/dialogs/boards/permissions/{pkCategoryHandle}', '\Concrete\Controller\Dialog\Board\Permissions::view');
+
+/* Permissions Tools Hack */
+$router->all('/tools/required/permissions/categories/board_admin', '\Concrete\Controller\Board\Permissions::process');
+$router->all('/tools/required/permissions/categories/board', '\Concrete\Controller\Event\Permissions::processCalendar');
```

Comment:
@aembler Is it correct that the board "tool" is mapped to Event\Permissions::processCalendar?

Reply:
No this is wrong. There should probably be a processBoard method in the Permissions class that the other board permission uses.

Reply:
Fixed in https://github.com/concrete5/concrete5/pull/9108